src/main.rs

```diff
@@ -138,6 +138,7 @@ fn create(command: args::CreateCommand) -> Result<(), Error> {
     let genfstab = Tool::find("genfstab")?;
     let mkfat = Tool::find("mkfs.fat")?;
     let mkext4 = Tool::find("mkfs.ext4")?;
+    let mkdir = Tool::find("mkdir")?;
```

Comment:
No need external mkdir. Rust has [a function](https://doc.rust-lang.org/std/fs/fn.create_dir_all.html) for that.